config/config.go

```diff
@@ -52,30 +51,32 @@ func setString(field reflect.Value, envVariable string, defaultValue string) {
 	if value == "" {
 		value = defaultValue
 		log.Printf("%s: Using default [%s].", envVariable, defaultValue)
+	} else {
+		log.Printf("%s: Found setting [%s].", envVariable, value)
 	}
-	log.Printf("%s: Found setting [%s].", envVariable, value)
 	field.SetString(value)
 }
 
 func setBool(field reflect.Value, envVariable string, defaultValue string) {
-	// defaultValue, _ := strconv.ParseBool(defaultValueAsString)
 	value := os.Getenv(envVariable)
-	if value == "" || strings.ToUpper(value) == "FALSE" {
-		log.Printf("%s: Using default [FALSE].", envVariable)
-		field.SetBool(false)
+	if value == "" {
+		value = defaultValue
+		log.Printf("%s: Using default [%s].", envVariable, value)
+	} else {
+		log.Printf("%s: Found setting [%s].", envVariable, value)
 	}
-	log.Printf("%s: Found setting [%s].", envVariable, value)
-	field.SetBool(true)
+	boolVal, _ := strconv.ParseBool(value)
```

Comment:
I suspect this isn't an issue at this point, but do we need to verify the value sent?

"ParseBool returns the boolean value represented by the string. It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False. Any other value returns an error. "